hack/local-up-cluster.sh

```diff
@@ -681,6 +716,9 @@ EOF
 fi
 }
 
+# let's check that etcd is in path before moving on
```

Comment:
yeah, making a separate `kube::etcd::validate` sounds good to me. (might want to have `kube::etcd::start` call it to maintain existing behavior.)